src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -77,7 +78,7 @@
     mode: 0400
     owner: root
     group: root
-  when: not fake_ssl_cert and nginx_combined_cert_value
+  when: not fake_ssl_cert and nginx_combined_cert_value is defined and nginx_combined_cert_value != ''
```

Comment:
could just be ` and nginx_combined_cert_value|default(None)`. This will also remove the need for the `is defined` check.